src/quote/Quote.ts

```diff
@@ -77,6 +104,15 @@ export function createAttesterSignature(
   }
 }
 
+/**
+ * Creates a [[Quote]] object with a given [[Identity]].
```

Comment:
Suggestion:
Maybe you can precise that this creates a quote object **signed** by the given identity.

Reply:
Agreed